code/game/machinery/embedded_controller/embedded_controller_base.dm

```diff
@@ -50,16 +50,6 @@
 	if(istype(signal) && program)
 		program.receive_signal(signal)
 
-/obj/machinery/embedded_controller/Topic(href, href_list)
-	. = ..()
-	if(.)
-		return
-
-	process_command(href_list["command"])
-
-	usr.set_machine(src)
-	addtimer(CALLBACK(src, .proc/updateDialog), 5)
```

Comment:
This is not dead code. The command prop is still used 
![image](https://user-images.githubusercontent.com/58055496/160259289-860151a9-64e6-40af-ac92-961efa3def79.png)